src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -10003,20 +10004,34 @@ void StorageReplicatedMergeTree::backupData(
 
     /// Send a list of mutations to the coordination too (we need to find the mutations which are not finished for added part names).
     {
-        auto zookeeper = getZooKeeper();
-        Strings mutation_ids;
-        if (zookeeper->tryGetChildren(fs::path(zookeeper_path) / "mutations", mutation_ids) == Coordination::Error::ZOK)
+        std::vector<IBackupCoordination::MutationInfo> mutation_infos;
+        ZooKeeperRetriesControl retries_ctl("getMutations", zookeeper_retries_info, local_context->getProcessListElement());
+        retries_ctl.retryLoop([&]()
         {
-            std::vector<IBackupCoordination::MutationInfo> mutation_infos;
-            mutation_infos.reserve(mutation_ids.size());
-            for (const auto & mutation_id : mutation_ids)
+            /// FIXME: should we retry tryGetChildrent and tryGet separately?
```

Comment:
One potential problem about retrying the whole loop is that if the mutation list is 100 elements and it fails at 50 we will retry the whole 100 elements (instead of just the missing 50). We might want to consider different approaches in the future

Reply:
Agree, I added the FIXME comment as I was not sure we can retry them separately from the `tryGetChildren` request, so decided to wait for @vitlibar  here to clarify.